server/data/forum.ts

```diff
@@ -31,7 +31,7 @@ export const getForumPosts = async (
 export const createForumPost = async (userId: string,
      content: string): Promise<ForumPostObject> => {
```

Comment:
Also do !content || content.trim() == ""

Reply:
I added !content, but content can be empty if they want so we don't need to check if it's empty